### PR TITLE
Proposed update to README.md

### PR DESCRIPTION
**Pull request by davidbayley80**
- An example HTML doc would make this much easier for noobs (like me!) to get the code up and running in a standalone browser. 
- Added 'league-container' as requested. 
- Modified the 'renderLeague' example. As you only added 4 teams in the previous 'addTeam' example,  you can't render the league table without either adding more teams or modifying the promotion/relegation positions. I've therefore changed the dropdown to 3, and the promotion and relegation zones to 1 and 4, respectively.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ Load the files straight into your HTML from the ```dist``` folder:
 <link rel="stylesheet" href="soccer.min.css">
 <script src="soccer.min.js"></script>
 ```
+An example html document is included in the distribution as an example.
```
Comment (jrobind) [inline README.md]: Do you mean within `/docs`?

File: README.md [hunk 2]
```diff
@@ -27,7 +29,9 @@ require(['soccer'], function (soccer) {
 Make sure you add a league table container:
 
 ```html
-<div class="league-table"></div>
+  <div class="league-container">
```
Comment (jrobind) [inline README.md]: I've just checked this again, we actually only need `<div class="league-table"></div>` and not the wrapping container div.

**Review by jrobind — COMMENTED** [review]
Thanks for the pull request! Helping to improve the documentation is super useful and much appreciated.

Looks like we no longer need the wrapping container div, but good spot regarding the zones and dropdown config within `renderLeague()` – the proposed updates look great! 

I've left a couple of comments, if you can review them and make a few changes I can approve and merge.

Thanks again!

File: README.md (hunk 1 above)
Comment (davidbayley80) [reply README.md]: Yes, a HTML containing the relevant markup and classes.

Comment (davidbayley80) [reply README.md]: Happy to pick this up btw^

Comment (jrobind) [reply README.md]: So, at the moment within `/docs` there is already an html file which is for https://jrobind.github.io/soccer.js/ (`/docs` is used for GitHub pages). I'm not sure how much additional value adding another file will bring. Maybe creating a UI to interact with the library would be a cool thing to do? Let me know if you'd be happy to take this on!

Comment (jrobind) [reply README.md]: If you remove this comment and also the comment regarding the unnecessary wrapping div below, I can merge this!

Comment (davidbayley80) [reply README.md]: > So, at the moment within `/docs` there is already an html file which is for https://jrobind.github.io/soccer.js/ (`/docs` is used for GitHub pages). I'm not sure how much additional value adding another file will bring. Maybe creating a UI to interact with the library would be a cool thing to do? Let me know if you'd be happy to take this on!

I actually didn't realise that the HTML within the /docs was interactive lol. Agreed, no need for a further example :)

Comment (davidbayley80) [reply README.md]: > If you remove this comment and also the comment regarding the unnecessary wrapping div below, I can merge this!

That should be done now (slowly learning how to use GitHub properly!)

Comment (davidbayley80) [reply README.md]: I'd be happy to take up adding a UI functionality for the app. Sounds like a good first js project to get my teeth stuck into. 

Would you recommend glitch as an IDE?

Comment (jrobind) [reply README.md]: Glitch is cool, totally up to you! I use a text editor called VS Code which is pretty popular with frontend devs.

**Review by jrobind — APPROVED** [review]
Looks great, thanks for the updates!

**Pull request merged**